### DISCUSSION TITLE
Camera Capture:  Add Interval Timer Sound

### DISCUSSION THREAD

**Discussion by motoglaf** [opentoonz/opentoonz#6480] [Ideas]
### Is your feature request related to a problem? Please describe

Could we get a sound for the interval timer in camera capture? That way we will know by maybe use of beeps for a count down and a final beep or shutter noise when the shot is completed. Stopmotion the animation app has this and it's very helpful so you don't have to keep looking at the screen especially if you have hundreds of drawings to shoot back to back with only one person shooting them.

### Describe the solution you'd like

Would like to have an implantation for a beep/sound for interval timer. The app stopmotion has a good example of how it works.

**Comment by RodneyBaker** [comment]
Hmmm... I actually thought there was an option for that in standard Camera Capture.
While awaiting such a fewature to be added...

If we use the Stop Motion Controller (the other feature with camera capture) it has an Option (on the Options panel) to Play a sound on capture:

![image](https://github.com/opentoonz/opentoonz/assets/2576432/fba9821a-a150-423f-9ed6-8ff4d48f45b7)

**Comment by motoglaf** [comment]
The stop motion controller makes a sound, but it will not pick up my SLR on my down shoot rostrum like camera capture does. Tried everything to get it to work, it sees the camera but will not allow me to shoot. The camera capture menu works fine just no sound. Could we please just add this in camera capture setup?

Thanks,

**Comment by RodneyBaker** [comment]
Will tag this with 'roadmap' as the Tahoma2D fork does appear to have this interval timer sound option.
Some intrepid soul may find the code and port it but as of this moment no developer has expressed interest.
If the issue is that a specific camera is not being recognized then we are likely looking at a different problem.

Note that I must assume we are talking here specifically about the Camera Capture (Pencil Test) feature and not Stop Motion feature.  As such that might fall more firmly into the realm of it's author @shun-iwasawa.

![image](https://github.com/user-attachments/assets/2575d5f4-edae-42a1-90b3-27f90d0c78e7)

**Comment by RodneyBaker** [comment]
@motoglaf Interval Timer Sound should be available in the latest nightly release.
It might take a few years but we eventually get there.
Closing as implemented.
